kleros-sdk/src/dataMappings/utils/dataMappingTypes.ts

```
@@ -1,43 +1,54 @@
+  address: string; // Address of the contract emitting the event
+  args: any[]; // Function arguments
+  seek: string[]; // Call return parameters used to populate the template variables
+  populate: string[]; // Populated template variables
+};
+
 export type AbiEventMapping = {
   abi: string; // ABI of the contract emitting the event
   address: string; // Address of the contract emitting the event
   eventFilter: {
     // Event filter (eg. specific parameter value, block number range, event index)
     fromBlock: BigInt | string; // Block number range start
```

Biome

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)